bindata/assets/alerts/api-usage.yaml

```diff
@@ -34,7 +34,7 @@ spec:
               a successful upgrade to the next EUS cluster version with Kubernetes {{ $labels.removed_release }}.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: >-
-            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1.31"})
+            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1.3[23]"})
```

Comment:
@sanchezl how do i know what to put here ?

Reply:
Even OCP releases (odd Kube releases) are EUS, so next EUS release is +2 minor versions. 
Odd OCP releases (even Kube releases) are not EUS, so next EUS release is +1 minor version.
e.g.:
* OCP 4.18 (kube 1.31) is an EUS release so `APIRemovedInNextEUSReleaseInUse` should warn them of APIs removed in the next two releases: OCP 4.19(kube 1.32) and OCP 4.20 (kube 1.33) since it is possible to upgrade "directly" to OCP 4.20 (kube 1.33).

Reply:
thanks.